dali/test/python/conditionals/test_nests.py

```diff
@@ -20,13 +20,20 @@
 
 from nvidia.dali._conditionals import check_nesting_support
 
+import warnings
 
-if not check_nesting_support():
-    exit()
 
+def disable_for_incompatible_env(test_func):
+    def _skipped_test():
+        warnings.warn(f"{test_func.__name__} skipped due to lack of dm-tree for Python 3.6")
```

Comment:
Maybe it should raise SkipTest?

Reply:
Ok, I figured it out. It was due to the name of the placeholder function that was returned in decorator.

Now it reports as skipped